src/main/java/dev/roryclaasen/vcsparser/measures/ComputeNumAuthorsMetric.java

```diff
@@ -49,59 +53,92 @@ public void removeProjectCache(String projectKey) {
 			authorsCache.remove(projectKey);
 	}
 
+	protected void saveProjectCache(String projectKey, MetricDate date, String currentKey, Map<String, Boolean> authorMap) {
+		// Project (projectKey)
+		if (!authorsCache.containsKey(projectKey))

```

Comment:
I'm quite curious. Do you really need to index by projectKey? I thought you could never have 2 projects running at the same time at sonar. Perhaps only on commercial editions.

Reply:
I believe you can have more than one worker, however the reason is more due to the fact that post analysis the map needs to be cleaned so that there is no left over data ([`PostProjectAnalysisHook.java`](https://github.com/roryclaasen/sonar-vcsparser-plugin/tree/master/src/main/java/dev/roryclaasen/vcsparser)). I thought by indexing under project key would be easier as I am able to remove all in one call later on.

> #### Speed up your analysis by adding more Workers
> Increase the number of Compute Engine Workers with the Enterprise Edition.
>
> [Learn More](https://redirect.sonarsource.com/editions/enterprise.html)